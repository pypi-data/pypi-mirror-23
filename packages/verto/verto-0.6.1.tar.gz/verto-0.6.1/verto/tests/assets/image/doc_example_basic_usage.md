{image file-path="http://placehold.it/350x150" caption="Placeholder image" source="https://placehold.it/" hover-text="This is hover text" alignment="left"}
