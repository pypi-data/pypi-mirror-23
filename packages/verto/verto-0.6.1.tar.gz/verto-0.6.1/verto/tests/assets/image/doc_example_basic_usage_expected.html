<div>
<img class="left-align" src="http://placehold.it/350x150" title="This is hover text" />
<p>Placeholder image</p>
<p><a href="https://placehold.it/">Source</a></p>
</div>
